include/language_rules.h

```diff
@@ -0,0 +1,14 @@
+#ifndef HELLO_WORLD_COMPILER_LANGUAGE_RULES_H
+#define HELLO_WORLD_COMPILER_LANGUAGE_RULES_H
+
+#include <piranha.h>
+
+class LanguageRules : public piranha::LanguageRules {
+public:
+	LanguageRules();
+	~LanguageRules();
+
+	virtual void registerBuiltinNodeTypes();
+};
+
+#endif /* HELLO_WORLD_COMPILER_LANGUAGE_RULES_H */
```

Comment:
Always make sure files end in a newline. If this was in the original file then let me know so I can remove it in that repo.